src/utils/hooks/api/loans/utils.ts

```diff
@@ -26,9 +26,11 @@ const getPositionsSumOfFieldsInUSD = <T extends LendPosition | BorrowPosition>(
   positions.reduce((totalValue: Big, position: T) => {
     const price = getTokenPrice(prices, position.currency.ticker)?.usd;
 
+    // ray test touch <<
     if (price === undefined) {
       console.error(`useGetAccountCollateralization: No exchange rate found for currency: ${position.currency.name}`);
     }
```

Comment:
I think this is where the lending and borrowing is broken because of missing prices. Therefore this condition should be removed by refactoring properly.